packages/form-field/src/index.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import * as styled from './styles'
+import {useDefaults} from '@smashing/theme'
+import {useFormContext} from '@smashing/form'
+import '@smashing/theme'
+import {
+  FormFieldProps,
+  FormFieldLabelAppearance,
+  FormFieldAlertAppearance
+} from './types'
+
+const INVALID_CONTEXT_MESSAGE = `[@smashing/form-field] FormField can be used only in @smashing/form context:
+const {Form} = useForm({/* Your form config */})
+
+return (
+  <Form>
+    <FormField component={Input} name="email" />
+  </Form>
+)
+`
+
+export const FormField: React.FC<FormFieldProps> = ({
+  component,
+  description,
+  hint,
+  htmlFor,
+  id,
+  label,
+  name,
+  className,
+  ...props
+}) => {
+  const {Field, ErrorMessage} = useFormContext()
+  const {labelAppearance, alertAppearance, ...inputProps} = useDefaults(
+    'formField',
+    props,
+    {
+      labelAppearance: 'block' as FormFieldLabelAppearance,
+      alertAppearance: 'block' as FormFieldAlertAppearance
+    }
+  )
+
+  if (!ErrorMessage || !Field) {
```

Comment:
I'm thinking about possible scenarios when we would want to use this component independently, e.g. in a single field scenarios. Wouldn't we want to have an escape hatch where form context is optional and this control would be just a presentational component?

Reply:
Done, smashing/form is now optional